### ISSUE TITLE
error rate

### ISSUE THREAD

**Issue by christopher5106** [facebook/fb.resnet.torch#89]
Hi 
I've been trying your nets to see how they compete with my colleagues who choose Caffe framework. 
I've chosen Resnet-101 for the comparison.
I've able to go very close to their results by choosing carefully the learning rate, but I've not been able to beat them with the finetuned nets. 
Their top-1 error rate 19.5 % and best top-10 error rate  4.4%
My top-1: 21.833119826567 and top-5: 5.2932228025227 with fb.resnet.torch
How can I go beyond that results ?
Thanks a lot for your work.
Best

**Comment by colesbury** [comment]
Use whatever data augmentation and LR schedule your colleague is using?

There's not enough information in your post about training, data augmentation, or evaluation procedure to understand why you would get different results.

**Comment by christopher5106** [comment]
Hi Sam,
I couldn't get it work correctly. 
Exact caffe configuration would be something like

 -batchSize 12 -LRpolicy step -LR 0.01 -gamma 0.1 -stepsize 5000 -momentum 0.9 -weightDecay 0.0005. 

I've tested your recommandation on resnet 101 finetuned : 
-batchSize 128 -LR 0.1 -momentum 0.96 -weightDecay 0.0001

It does not converge as low, it stays above.

**Comment by colesbury** [comment]
The LR schedule I used (from the ResNet paper) is for batch size 256. If you decrease the batch size, you may have to adjust the learning rate.

**Comment by christopher5106** [comment]
Oh great. In which direction (increase or decrease) do you think? I'm not really sure about the implementation of the sgd optim.

**Comment by christopher5106** [comment]
Ok now I have a better score than my colleagues 👍
